test/integration/highlight_editor_spec.mjs

```diff
@@ -1841,14 +1848,18 @@ describe("Highlight Editor", () => {
             "ternative compilation technique for dynamically-typed languages"
           );
           const editorSelector = getEditorSelector(0);
-          const x = rect.x + rect.width / 2;
+          const x = Math.round(rect.x + rect.width / 2);
           let y = rect.y + rect.height / 2;
           await page.mouse.click(x, y, { count: 3, delay: 100 });
           await page.waitForSelector(editorSelector);
           await waitForSerialized(page, 1);
           await page.keyboard.press("Escape");
           await page.waitForSelector(`${editorSelector}:not(.selectedEditor)`);
 
+          const clickHandle = await waitForPointerUp(page);
```

Comment:
Why does this section have to be moved up in the test?

Reply:
I'm not sure it's useful... my idea was to make sure that the pointerover callback is set after the pointer has been moved at its initial position.